src/resources/postcss/resets/_enfold.pcss

```diff
@@ -0,0 +1,17 @@
+.tribe-theme-enfold {
+
+	/* Enfold is using the #top id in the <body> */
+	#top& {
+		.tribe-common {
+			input,
+			button,
+			select,
+			textarea {
```

Comment:
just checking that it's all 4 of these selectors that are affected, or if we can narrow it down to only the ones that are affected.